boot/zephyr/include/sysflash/sysflash.h

```diff
@@ -7,4 +7,8 @@
 #define FLASH_AREA_IMAGE_1 2
 #define FLASH_AREA_IMAGE_SCRATCH 3
 
+#define FLASH_AREA_IMAGE_2 FLASH_AREA_IMAGE_0
```

Comment:
Same story, this is just a Zephyr-specific copy of a Mynewt header; such a change would need to be discussed across OSes